ArduPlane/Parameters.cpp

```diff
@@ -1233,6 +1233,15 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Increment: 1
     // @User: Advanced
     AP_GROUPINFO("DSPOILER_AILMTCH", 21, ParametersG2, crow_flap_aileron_matching, 100),
+    
+    // @Param: ACRO_YAW_RATE
+    // @DisplayName: ACRO mode yaw rate, used by QACRO mode
+    // @Description: The maximum yaw rate at full stick deflection in ACRO mode
```

Comment:
Needs to say `QACRO` not `ACRO` as that will confuse a normal plane user as it won't have any effect.